src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -236,15 +236,15 @@ protected function endDot()
     /**
      * @internal
      */
-    protected function dotize($id)
+    protected function dotize(string $id)
     {
         return hash('sha1', $id);
     }
 
     /**
      * @internal
      */
-    protected function escape($value): string
+    protected function escape(string $value): string
     {
         return \is_bool($value) ? ($value ? '1' : '0') : addslashes($value);
```

Comment:
not sure about the type hint, give the code

Reply:
Good catch! Fixed